content-cache_rock/nginx_cfg.tmpl

```diff
@@ -20,7 +20,7 @@ server {{
         proxy_set_header X-Forwarded-Proto "";
         proxy_set_header X-Forwarded-Scheme "";
 
-        add_header X-Cache-Status "$upstream_cache_status from {JUJU_POD_NAME} ({JUJU_POD_NAMESPACE})";
+        add_header X-Cache-Status "upstream_cache_status from {JUJU_POD_NAME} {JUJU_POD_NAMESPACE}";
```

Comment:
Are you sure about the removal of `$` ?

Reply:
The pb is that this variable is an nginx variable and it will reply depending on the cache status of the required file.
So the `$` should remain in the rendered template otherwise nginx will concider it as a normal text and instead of responding with `hit` or `miss` it will simply put `uptream_cache_status` which is not what we expect